Formula/claws-mail.rb

```diff
@@ -1,8 +1,8 @@
 class ClawsMail < Formula
   desc "User-friendly, lightweight, and fast email client"
   homepage "https://www.claws-mail.org/"
-  url "https://www.claws-mail.org/releases/claws-mail-3.18.0.tar.gz"
-  sha256 "f107deec1f0debfae77f7f0022aef713f85a5b1f7d0b1a0e98f6eebe1e3556f2"
+  url "https://www.claws-mail.org/releases/claws-mail-4.0.0.tar.gz"
+  sha256 "6f04364715f97acd8c26c7edd61a57598d926f57e796c03bb9d9080cfa3dc894"
```

Comment:
claws-mail 4.1 has been released

```suggestion
  url "https://www.claws-mail.org/releases/claws-mail-4.1.0.tar.gz"
  sha256 "534603d60200f3be346830d2fafd90dbdf0b59b9b3b928bc0aa7f8c1c218ec31"
```